e2e/angular-extensions/src/cypress-component-tests.test.ts

```diff
@@ -14,8 +14,7 @@ import {
 import { names } from '@nx/devkit';
 import { join } from 'path';
 
-// TODO(leo): enable when https://github.com/cypress-io/cypress/pull/27030 is merged and released
-describe.skip('Angular Cypress Component Tests', () => {
+describe('Angular Cypress Component Tests', () => {
```

Comment:
There was another test I disabled because of this https://github.com/nrwl/nx/blob/master/e2e/cypress/src/cypress.test.ts#L181-L182.

Reply:
updated, thanks!